modules/ROOT/partials/configuration/highlight_on_focus.adoc

```diff
@@ -0,0 +1,34 @@
+[[highlight_on_focus]]
+== `+highlight_on_focus+`
+
+The `+highlight_on_focus+` option adds a blue outline to an instantiated {productname} editor when that editor is made the input focus.
+
+This allows users to clearly see when the editor is in focus, or which editor has focus if more than one {productname} instance is available.
+
```

Comment:
Do we have an `admone-requires-6.4v` file yet? One should go here

Reply:
To quote my commit message:

> Added include statement pointing to `admon-requires-6.4c.adoc`.